### PR TITLE
Fix reaction thickness after hover

### PR DESCRIPTION
**Pull request by AlbaLopez**

### CODE REVIEW

File: js/src/TooltipContainer.js [hunk 1]
```diff
@@ -93,6 +94,11 @@ function is_visible () {
  *                          'd' because gene labels do not have them.
  */
 function show (type, d) {
+  if(this.reactionsShown && this.reactionsShown != d) {
+    var reaction_size = this.reactionsShown.data ? this.map.scale.reaction_size(this.reactionsShown.data) : this.map.settings.get_option('reaction_no_data_size');
+    d3.select('#r' + this.reactionsShown.reaction_id).selectAll('.segment').transition().style('stroke-width', reaction_size).duration(100);
+  }
+  this.reactionsShown = d;
```
Comment (AlbaLopez) [inline js/src/TooltipContainer.js]: I tried to make a function to dont repeat the code, but I just have that the function does not exist, all the time.

Comment (matyasfodor) [reply js/src/TooltipContainer.js]: so if you look at the code, you'll see that unfortunately escher doesn't use classes. It defines plain functions in the module context, and binds them to the prototype. I think this is where you efforts fell short. To give you an example:
Here's the prototype definition at the top of the file: https://github.com/DD-DeCaF/escher/blob/95c919138fc30b0a7c3a3e5c54ad7570f7b292f4/js/src/TooltipContainer.js#L19
And here's the function definition: https://github.com/DD-DeCaF/escher/blob/95c919138fc30b0a7c3a3e5c54ad7570f7b292f4/js/src/TooltipContainer.js#L56

File: js/src/TooltipContainer.js [hunk 2]
```diff
@@ -121,9 +127,10 @@ function show (type, d) {
  * Hide the input.
  */
 function hide () {
-  var reaction_size = this.map.settings.get_option('reaction_no_data_size');
-  d3.selectAll('.segment').transition().style('stroke-width', reaction_size).duration(100);
-  this.placed_div.hide()
+  var reaction_size = this.reactionsShown.data ? this.map.scale.reaction_size(this.reactionsShown.data) : this.map.settings.get_option('reaction_no_data_size');
```
Comment (matyasfodor) [inline js/src/TooltipContainer.js]: Could you please wrap this line and the next one in multiple lines?

Comment (matyasfodor) [inline js/src/TooltipContainer.js]: Could you please wrap this line and the next one in multiple lines?

**Comment by matyasfodor** [comment]
Please also `git fetch upstream && git pull origin/master --rebase`. I fixed the tests on master.

**Pull request merged**